charts/portworx/v1.0.6/README.md

```diff
@@ -0,0 +1,76 @@
+# Portworx
+
+## **Pre-requisites**
+
+Use this Helm chart to deploy [Portworx](https://portworx.com/) and [Stork](https://docs.portworx.com/scheduler/kubernetes/stork.html) to your Kubernetes cluster.
+
+Prerequisites
+
+Refer to the [Install Portworx on Kubernetes via Helm](https://docs.portworx.com/portworx-install-with-kubernetes/install-px-helm/#pre-requisites) page for the list of prerequisites.
+
+## **Limitations**
+* The portworx helm chart can only be deployed in the kube-system namespace. Hence use "kube-system" in the "Target namespace" during configuration.
+
+## **Uninstalling the Chart**
+
+#### You can uninstall Portworx using one of the following methods:
+
+#### **1. Delete all the Kubernetes components associated with the chart and the release.**
+
+> **Note** > The Portworx configuration files under `/etc/pwx/` directory are preserved, and will not be deleted.
+
+To perform this operation simply delete the application from the Apps page
+
+#### **2. Wipe your Portworx installation**
+> **Note** > The commands in this section are disruptive and will lead to data loss. Please use caution..
+
+See more details [here](https://2.1.docs.portworx.com/portworx-install-with-kubernetes/install-px-helm/#uninstall)
+
+## **Documentation**
+* [Portworx docs site](https://docs.portworx.com/install-with-other/rancher/rancher-2.x/#step-1-install-rancher)
+* [Portworx interactive tutorials](https://docs.portworx.com/scheduler/kubernetes/px-k8s-interactive.html)
+
+## **Installing the Chart using the CLI**
+
+See the installation details [here](https://2.1.docs.portworx.com/portworx-install-with-kubernetes/install-px-helm/)
```

Comment:
same here. Let's just use docs.portworx.com instead of a specific version. Easier to maintain.